### PR TITLE
Add more ignore paths for AppVeyor and GitHub CIs

### PR DESCRIPTION
**Pull request by PBfordev**
There is probably many more individual files or even folders that could be added (but perhaps with little return value), so this change may not be complete but I hope it is a step in the right direction.

_<Removed the part about samples, both CIs built at least some samples in some configurations>_

### CODE REVIEW

**Comment by PBfordev** [comment]
I forgot to add the demos folder to GitHub CI. :(

I do not understand why this PR did not trigger AppVeyor CI, when it changes its configuration file?

File: appveyor.yml [hunk 1]
```diff
@@ -7,11 +7,25 @@ branches:
...
+    - src/gtk1/
+    - include/wx/motif/
+    - src/motif/
     - include/wx/osx/
     - src/osx/
+    - include/wx/unix/
+    - src/unix/
+    - include/wx/x11/
+    - src/x11/
+    - '*.md'
```
Comment (vadz) [inline appveyor.yml:28]:
```suggestion
    - '**/*.md'
```
maybe?

**Comment by vadz** [comment]
> I forgot to add the demos folder to GitHub CI. :(

I'll just add it without pushing to avoid rerunning the CI builds once again.
 
> I do not understand why this PR did not trigger AppVeyor CI, when it changes its configuration file?

Sorry, no idea neither, and I couldn't find any way to debug it.

**Comment by PBfordev** [comment]
I think perhaps the `locale` folder could be added as well. It does not change often but AFAICT, it never requires a CI action. Not sure about the art `folder`.

**Comment by vadz** [comment]
Thanks, I've added `locale` to the list of ignored paths, but I'm less sure about `art`, so I didn't do anything for it -- especially because commits changing (only) the files in this directory are very rare.

**Pull request merged**